edx_proctoring/backends/backend.py

```diff
@@ -63,6 +65,22 @@ def get_javascript(self):
         """
         return ""
 
+    def get_ping_failure_should_block_user(self):
```

Comment:
This isn't really part of the public backend api. Since it's only used in one place, I think the logic of checking the waffle flag should move to the caller. That would save some testing, complexity, and coverage.